### PR TITLE
Consumer connected only to the followers

### PR DESCRIPTION
**Pull request by Gsantomaggio**
Fixes https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/issues/351

### CODE REVIEW

**Comment by Gsantomaggio** [comment]
@francescopessina87 do you have an opportunity to test it?

File: RabbitMQ.Stream.Client/RoutingClient.cs [hunk 1]
```diff
@@ -180,8 +186,14 @@ await LookupConnection(clientParameters, metaDataInfo.Leader, MaxAttempts(metaDa
         public static async Task<IClient> LookupRandomConnection(ClientParameters clientParameters,
```
Comment (francescopessina87) [inline RabbitMQ.Stream.Client/RoutingClient.cs:186]: @Gsantomaggio could it be a good idea to rename this method? We are no longer looking up for any "random" connection, but for a random "replica" connection (if any) or a "leader" connection... is that significant for the callers of this method?

Comment (francescopessina87) [inline RabbitMQ.Stream.Client/RoutingClient.cs:186]: @Gsantomaggio I would also correct the comment specifying the new behaviour

Comment (Gsantomaggio) [reply RabbitMQ.Stream.Client/RoutingClient.cs]: done via https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/pull/352/commits/cfb6b241d4007a65d9ebf84d717f6afddfdd5b76

Comment (Gsantomaggio) [reply RabbitMQ.Stream.Client/RoutingClient.cs]: done via https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/pull/352/commits/cfb6b241d4007a65d9ebf84d717f6afddfdd5b76

**Review by francescopessina87 — APPROVED** [review]

**Comment by Gsantomaggio** [comment]
Thank you @francescopessina87 ! Let's merge it!

**Pull request merged**